lib/block_view_validator.go

```diff
@@ -1660,6 +1603,11 @@ func (bav *UtxoView) IsValidRegisterAsValidatorMetadata(
 		metadata.DisableDelegatedStake { // Updating DisableDelegatedStake = true
 
 		// Retrieve existing StakeEntries.
+		//
+		// TODO: This is currently a very inefficient way to check to see if a validator has any
```

Comment:
we probably should address this now

Reply:
Addressed in https://github.com/deso-protocol/core/pull/531.